ios/magnet/ScannerBle.swift

```diff
@@ -9,19 +9,18 @@
 import Foundation
 
 @objc(ScannerBle) class ScannerBle: NSObject, BeaconScannerDelegate {
-  
   var bridge: RCTBridge!
   var scanner: BeaconScanner!
   
   @objc func start() -> Void {
-    NSLog("starting scanner");
-    self.scanner = BeaconScanner();
+    print("starting scanner")
```

Comment:
I liked the part where the logs are grouped and different threads wont disturb and mix. But if Emily, says to print, we print.